### DISCUSSION TITLE
Data Saturdays <> SQL Saturdays

### DISCUSSION THREAD

**Discussion by SQLDBAWithABeard** [dataplat/DataSaturdays#139] [Q&A]
How can we best ensure that this message is clear?

> Data Saturdays has been created as a place for the community create a set of tools and single point to enable people to run small regional events with little outlay or set-up diffculties to have open, transparent discussions on how it should be set up and utilised.

it basically provides a website and some support.

SQL Saturday assets have been bought and currently the messaging is unclear as to their future.

Data Saturdays is a community lead, open source, MIT licensed solution.

Suggestions how to make this message clearer please.

**Comment by way0utwest** [comment]
I think some confusion is unavoidable, at least until Redgate releases some direction. I hope that will come soon, but I have no timeline.

In terms of Data Saturday, I think it would behoove this group to better define what we do or intend to do. Assume that SQL Saturday, Sharepoint Saturday, et al, will exist. What exactly does this group do and what is available. Are tools here for integrating a site into Sessionize available for non-Data Saturdays? We have listed a few already, without nearly enough debate or discussion on which events we list. Do we list SQL Saturday events, becoming an hierarchy above them and others? Do we share tools with SQL Sat et al? Do we just become a hub for listing local events?

As someone that straddles both worlds, I am very unsure of what we should do here.
